lib/hammer_cli_foreman/auth_source.rb

```diff
@@ -18,16 +19,6 @@ class ListCommand < HammerCLIForeman::ListCommand
       build_options
     end
 
-    class UpdateCommand < HammerCLIForeman::UpdateCommand
-      desc _('Update organization and location for Auth Source')
-      resource :auth_source_externals
-  
-      success_message _('Taxonomy updated.')
-      failure_message _('Taxonomy not changed. Please check if appropriate auth source exists')
-
-      build_options
-    end
-
```

Comment:
Removing this and created a new subcommand altogether, similar to the ldap subcommand. I hope this makes sense!

Reply:
DRYing out always makes sense :) But does it make sense to leave this command for backwards compatibility also?

If so, we could change the whole command definition to:
```ruby
    subcommand 'external', ...
    subcommand 'update', HammerCLIForeman::AuthSourceExternal::UpdateCommand.desc, HammerCLIForeman::AuthSourceExternal::UpdateCommand
    # ...
```

OR we leave this command as it is, but add the pretty the same line in the `HammerCLIForeman::AuthSourceExternal` command class definition.

Reply:
I added this command like 3 days ago and realized it was not the correct way, so i thought it would be ohkay not caring too much about it. Would that be ohkay for us or we want to do something like adding the subcommand. Let me know if you think otherwise :)